app.js

```diff
@@ -1,9 +1,14 @@
 const express = require("express");
-const { getTopics } = require("./controllers/topics.controllers.js");
+const {
+  getTopics,
+  getEndpoints,
+} = require("./controllers/topics.controllers.js");
 const app = express();
 
 app.use(express.json());
```

Comment:
Is this middleware needed at this stage where we only have a GET request? When do we need this middleware? We should only introduce new middleware, packages etc as and when we need them.